INSTALL.md

```diff
@@ -10,42 +10,52 @@
 
 Independently from the specific OS, make sure that the `gcc`, `g++`, `make`, `git`, and `libpng-dev` packages or their equivalents are installed and accessible to the development tools that are used by the project (this means that, for example, on Windows, the packages have to be installed in the WSL environment). The package names and installation methods may vary with each OS.
 
-Install the devkitARM toolchain of devkitPro as per [the instructions on their wiki](https://devkitpro.org/wiki/devkitPro_pacman). On Windows, follow the Linux instructions inside WSL as any steps about the Windows installer do not apply.
+# Requirements
 
-**Debian-based distro users:** This applies to Debian, Ubuntu, and similar distros, including in WSL. If necessary, install the `libarchive13`, `pkg-config`, and `gdebi-core` packages to be able to install devkitPro.
+Grab the latest version of [devkitPRO](https://github.com/devkitPro/pacman/releases/latest) from their GitHub that fits your CPU architecture (arm64 or amd64)
 
-**Windows 10 users:** WSL 2 is available in the 1903 release (build 18362) and later, therefore existing WSL 1 and [prerelease WSL](https://docs.microsoft.com/windows/wsl/install-legacy) users are recommended to update. Right-click the Start button or press `Win`+`X`, choose Run, and run `ms-settings:about` to determine the Windows version. Also check Windows Update to make sure your installation is up-to-date.
+**Windows 10 users:** [WSL 2][wsl] is available in the 1903 release (build 18362) and later, therefore existing WSL 1 and [prerelease WSL](https://docs.microsoft.com/windows/wsl/install-legacy) users are recommended to update. Right-click the Start button or press `Win`+`X`, choose Run, and run `ms-settings:about` to determine the Windows version. Also check Windows Update to make sure your installation is up-to-date.
 
-**Windows 7 and 8.1 users:** pret is no longer focusing on support in pokeemerald for [old versions of Windows](https://support.microsoft.com/help/13853) so consider upgrading to a current release of Windows 10 or try a third-party guide like [this one](https://www.pokecommunity.com/showthread.php?t=425246) instead.
+**Debian-based distro users (and Windows users running WSL v2):** This applies to Debian, Ubuntu, and similar distros, including in WSL.
 
+1. Run `sudo apt-get install gcc g++ make git libpng-dev libarchive13 pkg-config gdebi-core`. This grabs all the requirements to move forward
```

Comment:
```suggestion
1. Run `sudo apt-get install gcc g++ make git libpng-dev libarchive13 pkg-config gdebi-core libcapstone-dev`. This grabs all the requirements to move forward
```

I found that `make tools` also requires `libcapstone-dev` in order to complete. Is it worth adding in here?